server/src/modules/objects/objects.controller.ts

```diff
@@ -6,10 +6,13 @@ import {
   Body,
   Put,
   Delete,
+  UseGuards,
```

Comment:
objects are not created via REST, i think there is no need to implement this for controllers